docs/source/standard.md

```diff
@@ -113,65 +113,130 @@ The ability of the TRE organisation to provide and manage devices, workspaces, i
 
-GPU, FPGA, ASIC, xPU
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+The ability to provide accelerators such as GPUs
 
-#### 2.1.5 Cluster computing
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-accelerators
+* - Statement
+  - Guidance
+* - A TRE should provide access to accelerators such as GPUs if required by users.
+  - GPUs and pother accelerators are commonly used in machine learning and other computationally intensive research.
+    TREs should make it clear to users whether GPUs and other resources are available whilst projects are being assessed.
+```
 
-SLURM, Kubernetes etc.
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
-
-#### 2.1.6 Databases
+#### 2.1.5 Databases
 
+Provision of databases for users
 SQL, noSQL, etc.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-software-tools
+* - Statement
+  - Guidance
+* - A TRE may make data available to researchers using comonly used databases such as PostgreSQL, MSSQL, MongoDB, etc.
+  - Databases must be secured and only accessible to users within the same project.
+    If shared database servers are used database administrators must ensure the database enforces segregation of users.
+```
 
 ### 2.2 Infrastructure analytics
 
-The ability of the TRE organisation to process and analyse data about the usage of the TRE.
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+The ability of the TRE organisation to record and analyse data about the usage of the TRE.
+
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-software-tools
+* - Statement
+  - Guidance
+* - A TRE must record usage of the TRE.
+  - This may include the number of users, number of projects, the amount of data stored, number of datasets, the number of workspaces, etc.
+* - A TRE should record which datasets are accessed, and when
+  - This helps auditability of how sensitive data has been used
+* - A TRE should record computational resource usage at the user or aggregate level
+  - This is useful for optimising allocation of resources, and managing costs.
+```
 
 ### 2.3 Network management
 
 The ability of the TRE organisation to administer and secure network infrastructure using applications, tools and processes.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-software-tools
+* - Statement
+  - Guidance
+* - Networks must be managed and controlled to protect information in systems and applications
+  - Network infrastructure must prevent unauthorised access to resources on the network.
+    This may include firewalls, network segmentation, and restricting connections to the network.
+* - Networks must be continually monitored for misconfigurations and vulnerabilities
+  - This may include regular vulnerability scanning, and penetration testing.
```

Comment:
Overlap with 4.2.
Not necessarily a problem but it would be good to cross reference when we can.